src/ui/components/TestSuite/views/TestMetadata/Panel.module.css

```diff
@@ -18,7 +18,6 @@
   flex: 1 1 auto;
   font-size: var(--font-size-large);
   color: var(--body-color);
-  overflow: hidden;
```

Comment:
What motivated this change? It will cause a regression so I think it should be reverted.

### With overflow
![Screen Shot 2023-05-15 at 9 23 41 AM](https://github.com/replayio/devtools/assets/29597/9a97f1ef-6ce2-4146-b52b-f284f4456176)

### Without overflow
![Screen Shot 2023-05-15 at 9 23 32 AM](https://github.com/replayio/devtools/assets/29597/71ae968a-b8cd-44d1-80ac-682a005620a8)

Reply:
True. I was trying to address an issue where the descenders (for example, the lower part of the lowercase g) were getting cropped. This fixes that issue, but the side effect you point out is bad.

So instead I'm going to force a line height.

Reply:
If that was your concern, rather than forcing a line-height (which can cause other layout issues) – just change this to `overflow-x: hidden` ?